src/common/runner/runner.go

```diff
@@ -199,6 +204,10 @@ func (r *Runner) TagFile(file string) {
 						continue
 					}
 				}
+			}
+
+			if parser.Name() == "Dockerfile" {
```

Comment:
@sshintaku This means the `else` clause will be called only in the case of `Dockerfile`. Perhaps we can inverse the `if` and then handle the case of Dockerfile inside the new `else` clause?

Reply:
Let's try a different route. please make sure the `Block is not taggable` is available in case of non Dockerfile parsers @sshintaku

Reply:
Let's try a different route. please make sure the `Block is not taggable` is available in case of non Dockerfile parsers @sshintaku

Reply:
I am looking at this code and the way it is all non-docker files will have Block is not taggable available, do you not agree?  If you see problems with the logic, by all means let me know.

Reply:
Before this change, every untaggable block would be printed out. After this change, non-docker untaggable blocks will not be logged. Please fix @sshintaku